requirements/required.txt

```diff
@@ -9,7 +9,7 @@ lightly==1.5.13
 lightning[pytorch-extra]==2.4.0
 matplotlib==3.9.2
 numpy==2.1.1
-pandas==2.2.3
+pandas[parquet]==2.2.3
```

Comment:
I thought we got rid of this change, did this somehow come back? Parquet isn't required, only optional.

Reply:
I probably messed up the rebase